### PR TITLE
chore: pass localisation manifest URL in app config

### PR DESCRIPTION
**Pull request by blackrabbit99**
Application can optionally have link to localization manifest. So it can be used for example for handle immutable cache when your locale is dynamically updated

### CODE REVIEW

File: registry/client/src/dataProvider.js [hunk 1]
```diff
@@ -43,7 +43,9 @@ const myDataProvider = {
         params.id = encodeURIComponent(params.id);
 
         return dataProvider.getOne(resource, params).then(v => {
+            console.log(v);
```
Comment (b1ff) [inline registry/client/src/dataProvider.js:46]: seems like leftovers

File: registry/server/apps/routes/getApp.ts [hunk 2]
```diff
@@ -28,9 +28,13 @@ const getApp = async (req: Request<GetAppRequestParams>, res: Response): Promise
 
     const [app] = await db.select().from<App>('apps').where('name', appName);
 
+    console.log('app = ');
+    console.log(app);
```
Comment (b1ff) [inline registry/server/apps/routes/getApp.ts:32]: can be removed

File: registry/server/apps/routes/getApp.ts [hunk 3]
```diff
@@ -28,9 +28,13 @@ const getApp = async (req: Request<GetAppRequestParams>, res: Response): Promise
...
+
     if (!app) {
         res.status(404).send('Not found');
     } else {
+        console.log(preProcessResponse(app));
```
Comment (b1ff) [inline registry/server/apps/routes/getApp.ts:37]: also not needed I believe

File: registry/server/apps/routes/getApp.ts (hunk 2 above)
Comment (blackrabbit99) [reply registry/server/apps/routes/getApp.ts:32]: looks like fixed locally but w/o push :(

**Review by yehor-manzhula — APPROVED** [review]

**Review by b1ff — APPROVED** [review]

File: registry/server/apps/routes/getApp.ts (hunk 2 above)
Comment (blackrabbit99) [reply registry/server/apps/routes/getApp.ts]: solved

File: registry/server/apps/routes/getApp.ts (hunk 3 above)
Comment (blackrabbit99) [reply registry/server/apps/routes/getApp.ts]: fixed

**Pull request merged**